docs/usage/data-modeling.md

```diff
@@ -0,0 +1,150 @@
+# Data Modeling
+
+TigerBeetle is a domain-specific database — its schema of [`Account`s](../reference/accounts.md)
+and [`Transfer`s](../reference/transfers.md) is built-in and fixed. In return for this prescriptive
+design, it provides excellent performance, integrated business logic, and powerful invariants.
+This section is a sample of techniques for mapping an application's requirements onto TigerBeetle's
+data model. Which (if any) of these techniques are suitable is highly application-specific.
+
+When possible, encoding application invariants directly in TigerBeetle rather than implementing
+them in the application itself (or with a foreign database) minimizes round-trips and coordination.
```

Comment:
"encoding application invariants" and "minimizes round-trips and coordination" are far away from eachother which makes it harder to follow the sentence IMO

You could rewrite to:

When possible, round trips and coordination can be minimized by encoding ...